std/json.d

```diff
@@ -37,7 +37,7 @@ import std.traits;
 
     // check a type
     long x;
-    if (const(JSONValue)* code = "code" in j)
+    if (auto code = "code" in j)
```

Comment:
**const identifier** which now works breaks **const(Type)**. because it predates the typectors which are also used for the rule I broke. I'm gonna make a libdparse PR.